src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -245,13 +245,13 @@ const PreviewHeader = ({ queryParams, cluster, readOnlyAccess, onCreateCluster,
         ])
       ])],
       [clusterStatus === 'Creating', () => h(StatusMessage, [
-        'Creating notebook runtime, this will take 5-10 minutes. You can navigate away and return when it’s ready.'
+        'Creating notebook runtime environment. You can navigate away and return in 3-5 minutes.'
       ])],
       [clusterStatus === 'Starting', () => h(StatusMessage, [
-        'Starting notebook runtime, this may take up to 2 minutes.'
+        'Notebook runtime environment is stopped. Start it to edit your notebook or use the terminal.'
```

Comment:
This message doesn't seem to match the "Starting" state.

Reply:
SMH thanks, fixed it now